### PR TITLE
andThen() default method in BiFunction interface returning the wrong …

### PR DESCRIPTION
**Pull request by gcarravi**
…type and other small typos.

### CODE REVIEW

File: ch22.html [hunk 1]
```diff
@@ -1026,7 +1028,7 @@ <h2>Key Points</h2>
 
                 <li>If we create an instance of <code>ZonedDateTime</code> for a region where a Daylight Saving Time (DST) is observed, the instance will support it, advancing the clock one hour when DST starts, and setting it back when DST ends.</li>
 
-                <li><code>Period</code> and <code>Duration</code> differ in their treatment of DST.</li>
+                <li><code>Period</code> and <code>Duration</code> differ in their treatment of DST.</li>Predefined Formatters
```
Comment (eh3rrera) [inline ch22.html:1031]: I believe "Predefined Formatters" is out of place

**Comment by eh3rrera** [comment]
Hi @gcarravi, thank you so much for your changes. I left a comment on one line of the file ch22.html. Can you take a look to see if it applies, please?

**Comment by gcarravi** [comment]
Hi @eh3rrera. Your comment most certainly applies. Not sure how that happened. Sorry about it.
And since we are at it, I have noticed that I also missed line 804 in chapter 10:

<p><code class="java hljs"><span class="hljs-keyword">default</span> &lt;V&gt; Function&lt;T, V&gt; andThen(<br />
	  &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Function&lt;? <span class="hljs-keyword">super</span> R, ? extends V&gt; after)</code></p>


which again has the wrong return type and needs to be replaced with:

<p><code class="java hljs"><span class="hljs-keyword">default</span> &lt;V&gt; BiFunction&lt;T, U, V&gt; andThen(Function&lt;? <span class="hljs-keyword">super</span> R, ? extends V&gt; after)</code></p>

Just like we did for the BiFunction section.

Kind regards

Gemma

**Comment by eh3rrera** [comment]
Thank you so much Gemma! 👍

**Pull request merged**